metals/src/main/scala/scala/meta/internal/metals/codeactions/OrganizeImports.scala

```diff
@@ -0,0 +1,50 @@
+package scala.meta.internal.metals.codeactions
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+import scala.meta.internal.metals.CodeAction
+import scala.meta.internal.metals.MetalsEnrichments.XtensionString
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals.ScalafixProvider
+import scala.meta.pc.CancelToken
+
+import org.eclipse.lsp4j.CodeActionParams
+import org.eclipse.{lsp4j => l}
+
+final class OrganizeImports(scalafixProvider: ScalafixProvider)
+    extends CodeAction {
+  override def kind: String = OrganizeImports.kind
+
+  override def contribute(params: CodeActionParams, token: CancelToken)(implicit
+      ec: ExecutionContext
+  ): Future[Seq[l.CodeAction]] = {
+    if (isSourceOrganizeImportCalled(params)) {
```

Comment:
Could we check here if the file ends with `.scala` or `.sbt` ?

I don't think we need to show warnings worksheets or ammonite scripts. Code action should not be added in that case.

Reply:
ok